2021-01-26T15:27:15.4508597-04:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-01-26T15:27:15.4617571-04:00  [INF] Hosting environment: "Development" (c3307c92)
2021-01-26T15:27:15.4651251-04:00  [INF] Content root path: "C:\Users\Mayra\Source\Repos\JanWay_AWS\JWA.Server" (b5d60022)
2021-01-26T15:29:08.0859755-04:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-01-26T15:29:08.1149976-04:00  [INF] Hosting environment: "Development" (c3307c92)
2021-01-26T15:29:08.1178188-04:00  [INF] Content root path: "C:\Users\Mayra\Source\Repos\JanWay_AWS\JWA.Server" (b5d60022)
2021-01-26T15:34:11.9205393-04:00 80000023-0001-f700-b63f-84710c7967bb [ERR] Connection ID ""17798225733273780258"", Request ID ""80000023-0001-f700-b63f-84710c7967bb"": An unhandled exception was thrown by the application. (bf316047)
Npgsql.NpgsqlException (0x80004005): Severity not received in server error message
   at Npgsql.BackendMessages.ErrorOrNoticeMessage.Load(NpgsqlReadBuffer buf)
   at Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming)
   at Npgsql.NpgsqlDataReader.Consume(Boolean async)
   at Npgsql.NpgsqlDataReader.Close(Boolean connectionClosing, Boolean async)
   at Npgsql.NpgsqlConnector.CloseOngoingOperations(Boolean async)
   at Npgsql.NpgsqlTransaction.Dispose(Boolean disposing)
   at System.Data.Common.DbTransaction.Dispose()
   at Microsoft.EntityFrameworkCore.Storage.RelationalTransaction.Dispose()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Dispose()
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.Dispose()
   at Microsoft.EntityFrameworkCore.DbContext.Dispose()
   at JWA.Infrastructure.Repositories.UnitOfWork.Dispose() in C:\Users\Mayra\Source\Repos\JanWay_AWS\JWA.Infrastructure\Repositories\UnitOfWork.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.DisposeAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Http.Features.RequestServicesFeature.<DisposeAsync>g__Awaited|9_0(RequestServicesFeature servicesFeature, ValueTask vt)
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.FireOnCompleted()
2021-01-26T15:38:46.9475971-04:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-01-26T15:38:46.9608886-04:00  [INF] Hosting environment: "Development" (c3307c92)
2021-01-26T15:38:46.9642727-04:00  [INF] Content root path: "C:\Users\Mayra\Source\Repos\JanWay_AWS\JWA.Server" (b5d60022)
2021-01-26T15:39:16.3880752-04:00 80000024-0000-f900-b63f-84710c7967bb [ERR] Connection ID ""17942340917054668835"", Request ID ""80000024-0000-f900-b63f-84710c7967bb"": An unhandled exception was thrown by the application. (bf316047)
Npgsql.NpgsqlOperationInProgressException: The connection is already in state 'Executing'
   at Npgsql.NpgsqlConnector.<StartUserAction>g__DoStartUserAction|187_0(<>c__DisplayClass187_0& )
   at Npgsql.NpgsqlConnector.StartUserAction(ConnectorState newState, NpgsqlCommand command)
   at Npgsql.NpgsqlConnector.Rollback(Boolean async)
   at Npgsql.NpgsqlTransaction.Rollback(Boolean async)
   at Npgsql.NpgsqlTransaction.Rollback()
   at Npgsql.NpgsqlTransaction.Dispose(Boolean disposing)
   at System.Data.Common.DbTransaction.Dispose()
   at Microsoft.EntityFrameworkCore.Storage.RelationalTransaction.Dispose()
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Dispose()
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.Dispose()
   at Microsoft.EntityFrameworkCore.DbContext.Dispose()
   at JWA.Infrastructure.Repositories.UnitOfWork.Dispose() in C:\Users\Mayra\Source\Repos\JanWay_AWS\JWA.Infrastructure\Repositories\UnitOfWork.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngineScope.DisposeAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Http.Features.RequestServicesFeature.<DisposeAsync>g__Awaited|9_0(RequestServicesFeature servicesFeature, ValueTask vt)
   at Microsoft.AspNetCore.Server.IIS.Core.IISHttpContext.FireOnCompleted()
